src/applications.md

```diff
@@ -13,6 +13,7 @@ Applications of Rust:
 * [intermezzOS OS](http://intermezzos.github.io/book/second-edition/)
 * [Juniper - GraphQL Server for Rust](https://graphql-rust.github.io/juniper/current/)
 * [mdBook](https://rust-lang.github.io/mdBook/)
+* [nalgebra](https://nalgebra.org/docs/)
```

Comment:
Hmmm. Nice book, but since it isn't in mdbook format (see https://github.com/dimforge/nalgebra.org), it belongs in the "other formats" chapter.